index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aimal Amiri</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav class="container">
+        <a href="#" class="navbar-logo font-inter text-bold decoration-none">Aimal Amiri</a>
+        <ul class="navbar-menu">
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">Portfolio</a></li>
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">About</a></li>
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">Contact</a></li>
+            <li class="navbar-link font-inter">
+                <a href="#" class="text-white decoration-none">
+                    <img src="./assets/svg/mail.svg" alt="mail" class="text-white" />
+                </a>
+            </li>
+        </ul>
+    </nav>
+    <header class="header">
+        <div class="social-media-links">
+            <a href="#" class="btn btn-circle"><img src="assets/svg/github.svg" alt="Github icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/linkedin.svg" alt="Linkedin icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/angel-list.svg" alt="Angel List icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/twitter.svg" alt="Twitter icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/medium.svg" alt="Medium Icon"></a>
+        </div>
+        <section class="header-content container">
+            <h1 class="header-title font-crete-round text-orange">Hey There. I'm Aimal</h1>
+            <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2>
+            <p class="header-text text-white font-inter">I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </section>
+        <img class="header-image" src="./assets/png/header-illustration.png" alt="Header illustration Photo">
```

Comment:
Please ensure that you use `css` background image (decorations are always background images, not `img` tags)